services/src/main/java/org/keycloak/services/clientpolicy/context/DeviceAuthorizationRequestContext.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.services.clientpolicy.context;
```

Comment:
@tnorimat Is it possible that stuff related to device grant is wrapped in the single place in the codebase whenever possible? Similarly like we try to do for CIBA for example. Hence it is possible to move this class to some sub-package under "org.keycloak.protocol.oidc.grants.device" ?

Reply:
@mposolda Yes, I will do that.